option optimize_for = LITE_RUNTIME;

message BuyWudiBossBuffRequest
{
 required uint32 index = 1;
 required uint32 boss_ts = 2;
 required uint32 buff_type = 3;
 required uint32 buff_index = 4;
}
